src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,42 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const defaultSelectedTab = tabs.find(
+    tab => tab.id === selectedTabId,
+  ) || tabs[0];
+
+  const handleTabClick = (tab) => {
+    if (tab.id !== selectedTabId) {
+      onTabSelected(tab);
+    }
+  };
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              key={tab.id}
+              data-cy="Tab"
+              className={tab.id === defaultSelectedTab.id
+                ? 'is-active'
+                : ''
+              }
+            >
+              <a
+                href={`#${tab.id}`}
+                data-cy="TabLink"
+                onClick={() => handleTabClick(tab)}
+              >
+                {tab.title}
+              </a>
+            </li>
+          ))}
+        </ul>
+      </div>
+
+      <div className="block" data-cy="TabContent">
+        {tabs.find(tab => tab.id === defaultSelectedTab.id)?.content}
```

Comment:
This `find` is weird - why do it again, if you already have selected tab?

Also minor thing, but this is the only place I would actually use `defaultTab` - in all other I suggest using just `selectedTabId` for consistency